pkg/system/names.go

```diff
@@ -1,33 +0,0 @@
-/*
-Copyright 2019 The Tekton Authors
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package system
-
-import "os"
-
-const (
-	DefaultNamespace      = "tekton-pipelines"
```

Comment:
the `knative/pkg` system package doesn't have this `DefaultNamespace`, so this *might* break users if the are running the controller without `SYSTEM_NAMESPACE`.

Reply:
can we set SYSTEM_NAMESPACE to a default in the config yamls maybe?

Reply:
It is set here:
```yaml
        env:
        - name: SYSTEM_NAMESPACE
          valueFrom:
            fieldRef:
              fieldPath: metadata.namespace
```

Reply:
https://github.com/tektoncd/pipeline/blob/master/config/controller.yaml#L80

So this would really only present an issue if people ran with modified YAMLs.

Reply:
@dlorenc yeah that was my thought. It's a really *niche* problem as I think the number of user / integrator is really small. Most likely having a big warning in the release note should be enough.

Reply:
Added a section in the release notes.